tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/get-started-experience.cy.js

```diff
@@ -0,0 +1,44 @@
+// <reference types="Cypress" />
+
+describe('Start Setup WP Experience Page', function () {
+
+	before(() => {
+		cy.visit('wp-admin/?page=nfd-onboarding&flow=ecommerce#/wp-setup/step/get-started/experience');
+		cy.injectAxe();
+	});
+
+    it('Check if Drawer Suppressed', () => {
+        cy.get('.nfd-onboarding-drawer__panel-site-title-container').scrollIntoView().should('not.be.visible');
+    });
+
+    it('Check if Headers Load', () => {
+        cy.get('.nfd-step-card-heading').should('exist');
+        cy.get('.nfd-step-card-subheading-other').should('exist');
+        cy.get('.nfd-step-card-question').should('exist');
+    });
+
+    it('Check if radio options load', () => {
+        cy.get('.components-radio-control__option').should('exist');
+    });
+
+    it('Navigation Buttons Landing on expected pages', () => {
+        cy.get('button').contains('Back').click();
+        cy.url().should('contain', 'get-started/welcome');
+        cy.get('button').contains('Next').click();
+        cy.url().should('contain', 'get-started/experience');
```

Comment:
We might not have these as the next and back steps in the future, so to handle this the only thing we need to ensure is that on Next and Back the url is not #/wp-setup/step/get-started/experience. This test is also failing for me.

Reply:
Made the modifications accordingly, and the testcases are passing